go/uidmap/hardcoded_uids.go

```diff
@@ -21,7 +21,7 @@ var usernameSortOrder []uint16
 var usoOnce sync.Once
 
 func findInit() {
-	once.Do(func() {
+	usoOnce.Do(func() {
```

Comment:
can you use `once` here and `usoOnce` in the other init?  `uso` = "usernameSortOrder"